api/apps/users/model.coffee

```diff
@@ -65,10 +65,17 @@ save = (user, accessToken, callback) ->
       request
         .get("#{ARTSY_URL}/api/v1/profile/#{user.default_profile_id}")
         .set('X-Access-Token': accessToken).end cb
+    if user.type is "Admin"
+      (cb) ->
+        request
+        .get("#{ARTSY_URL}/api/v1/me/authentications")
+        .set('X-Access-Token': accessToken).end cb
   ], (err, results) ->
     return callback err if err
     encryptedAccessToken = results[0]
     profile = results[1].body
+    facebook_uid = results[2].body[0].uid if results[2].body[0].uid?
```

Comment:
Code golf tip you can assign a var in your if like so `facebook_uid = uid if uid = results[2].body[0].uid?`